### PR TITLE
Update cron library for ldap sync

### PR DESCRIPTION
**Pull request by Jguer**
**What this PR does / why we need it**:

https://github.com/grafana/grafana-enterprise/pull/3193

- Removes cron (v1) dependency without breaking previous configurations
- Nudge new users to use standard cron notation

>  The v1 branch accepted an optional seconds field at the beginning of the cron spec. This is non-standard and has led to a lot of confusion. The new default parser conforms to the standard as described by [the Cron wikipedia page](https://en.wikipedia.org/wiki/Cron).

https://github.com/robfig/cron#upgrading-to-v3-june-2019=

### CODE REVIEW

**Review by IevaVasiljeva — APPROVED** [review]
What would happen to users who have set up their cron configuration using the non-standard format (as in, specifying seconds)? Would Grafana still be able to parse it, or would it break things?

If it wouldn't cause issues to these users, then it looks good to me! 🚢

**Comment by Jguer** [comment]
> What would happen to users who have set up their cron configuration using the non-standard format (as in, specifying seconds)? Would Grafana still be able to parse it, or would it break things?
> 
> If it wouldn't cause issues to these users, then it looks good to me! ship

Grafana will still parse it and operate as usual when seconds are specified, they've been made optional now instead of required

File: docs/sources/enterprise/enhanced_ldap.md [hunk 1]
```diff
@@ -48,9 +48,9 @@ Removed users are automatically logged out and their account disabled. These acc
 # @weekly                | Run once a week, midnight between Sat/Sun  | 0 0 0 * * 0
 # @daily (or @midnight)  | Run once a day, midnight                   | 0 0 0 * * *
 # @hourly                | Run once an hour, beginning of hour        | 0 0 * * * *
-sync_cron = "0 0 1 * * *" # This is default value (At 1 am every day)
+sync_cron = "0 1 * * *" # This is default value (At 1 am every day)
 # This cron expression format uses 6 space-separated fields (including seconds), for example
```
Comment (Jguer) [inline docs/sources/enterprise/enhanced_ldap.md:52]:
```suggestion
# This cron expression format uses 5 space-separated fields, for example
```

**Comment by IevaVasiljeva** [comment]
> > What would happen to users who have set up their cron configuration using the non-standard format (as in, specifying seconds)? Would Grafana still be able to parse it, or would it break things?
> > If it wouldn't cause issues to these users, then it looks good to me! ship
> 
> Grafana will still parse it and operate as usual when seconds are specified, they've been made optional now instead of required

Gotcha, sounds good!

**Pull request merged**